server/website/website.go

```diff
@@ -72,9 +73,11 @@ func GetContent(websiteName string, path string) (string, []byte, error) {
 
 	dbSession := db.Session()
 	content := models.WebContent{}
+	// Use path without any query parameters
+	u, _ := url.Parse(path)
```

Comment:
We should not discard the error here, it's unlikely to happen but just in case, check whether `url.Parse` succeeded.